src/server/main.ts

```diff
@@ -56,7 +56,10 @@ async function bootstrap(): Promise<void> {
         swaggerOptions: {
             persistAuthorization: false,
             initOAuth: {
-                clientId: keycloakConfig.CLIENT_ID,
+                clientId:
+                    process.env['DEPLOY_STAGE'] === 'dev' || process.env['NODE_ENV'] === 'dev'
+                        ? keycloakConfig.TEST_CLIENT_ID
+                        : keycloakConfig.CLIENT_ID,
```

Comment:
On the cluster both of these environment variables are set to "prod", even for our test-branches (Like the one for this PR).